build-system/extensions-versions-config.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the license.
+ */
+
+var extensionsVersions = {
+  'amp-sticky-ad': ['1.0'],
+};
```

Comment:
Yes! That would be an approach, and I did think about it previously.  But I figure if later we are going to have multiple versions, we can be more flexible and make the config looks like:
```
{
   extensionName: {
     'latest': '3.0'
     'support': ['2.0']
     'deprecated': ['1.0']
     'invalid': ['0.1']
  }
}
```
For example in amp-sticky-ad case, 0.1 is deprecated, but it's still valid. I think it's best we keep a record of all previous version at some place. Or if you think it's not necessary, I am OK to remove this file. But then I'll have to remove 0.1 folder in this PR as well. 
Let me know